django/contrib/admin/filters.py

```diff
@@ -176,8 +176,17 @@ def __init__(self, field, request, params, model, model_admin, field_path):
         self.title = self.lookup_title
         self.empty_value_display = model_admin.get_empty_value_display()
 
+    @property
+    def relation_allows_none(self):
```

Comment:
Perhaps something like `allows_not_any`, `allows_empty_choice`?

The distinction here isn't "this has a python `None` value", it's a "none of these are included, because the relation is an empty set".

Perhaps a docstring along those lines would make sense here, too. Eg.

    """
    Return `True` if a "(None)" choice should be included,
    which filters out everything except empty relationships.
    """